src/mono/browser/runtime/exports.ts

```diff
@@ -70,6 +70,10 @@ function initializeExports(globalObjects: GlobalObjects): RuntimeAPI {
         runtimeList = globalThisAny.getDotnetRuntime.__list;
     }
 
+    if (BuildConfiguration === "Debug") {
+        globalThisAny.INTERNAL = globals.internal;
```

Comment:
Could you please add an explanation?
Would this break our test with multiple runtimes?
Could it based on some parameter?

Reply:
BuildConfiguration === "Debug"

Reply:
anyway, I will remove it